src/module/actor/actor-inventory-utils.js

```diff
@@ -247,7 +247,9 @@ export async function moveItemBetweenActorsAsync(sourceActor, itemToMove, target
         }
 
         /** Ensure the original to-move item has the quantity correct. */
-        itemData[0].data.quantity = quantity;
+        if (itemData[0].system?.quantity) { //Check for V10
```

Comment:
This is problematic at the moment. The goal for this update is to not break v9 comparability. We'll need to make use of a version check here to determine how this block of code needs to be handled. I don't like doing it, but it's gonna be the only way to handle v9 and v10 compatibility for this one.